tmk_core/protocol/chibios/usb_main.c

```diff
@@ -683,20 +715,35 @@ static bool usb_request_hook_cb(USBDriver *usbp) {
 #if defined(SHARED_EP_ENABLE) && !defined(KEYBOARD_SHARED_EP)
                             case SHARED_INTERFACE:
 #endif
+#if defined(MOUSE_ENABLE) && !defined(MOUSE_SHARED_EP)
+                            case MOUSE_INTERFACE:
```

Comment:
this needs an #endif ?

After I added that from the latest commit, the firmware compiles (using an RP2040)

Reply:
yup, thanks will add to upcoming commit!
Really need to set up a dummy RP2040 build to test compiles or maybe I could just try compiling your board, Scylla something right?